autosave/mixins.py

```diff
@@ -1,27 +1,23 @@
+from __future__ import absolute_import
 import time
 import json
 import functools
 import textwrap
 from datetime import datetime
-from urlparse import urlparse
+from six.moves.urllib.parse import urlparse
```

Comment:
I notice that `six` isn't in `install_requires`, only `Django`. Should we import six from `django.utils.six` instead?

Reply:
Good catch, I've added `django.utils`